cmd/commands/dlv/dlv_amd64.go

```diff
@@ -28,10 +28,10 @@ import (
 	"github.com/beego/bee/cmd/commands/version"
 	beeLogger "github.com/beego/bee/logger"
 	"github.com/beego/bee/utils"
-	"github.com/derekparker/delve/service"
-	"github.com/derekparker/delve/service/rpc2"
-	"github.com/derekparker/delve/service/rpccommon"
-	"github.com/derekparker/delve/terminal"
+	"github.com/go-delve/delve/service"
+	"github.com/go-delve/delve/service/rpc2"
+	"github.com/go-delve/delve/service/rpccommon"
+	"github.com/go-delve/delve/terminal"
```

Comment:
The `github.com/go-delve/delve/terminal` dose not exist in project `github.com/go-delve/delve` .